bin/migrate_tables.py

```diff
@@ -50,6 +54,17 @@ def run_subprocess(command: str, shell: bool = True):
 def migrate_table(
     source_table: TableRef, dest_table: TableRef, username: str = None, dryrun: bool = False
 ):
+    # Ensure destination table does not exist already (if it does there may be unexpected results
+    # from the upload command, e.g. duplicate records)
+    with Postgres(host=dest_table.host, database=dest_table.database, user=username) as db_src:
+        dest_table_exists = db_src.table_or_view_exists(dest_table.table, schema=dest_table.schema)
+        if dest_table_exists:
+            logger.error(
+                f"Destination table exists, skipping upload: {dest_table.schema}.{dest_table.table}"
```

Comment:
Probably not worth it, but it would be nice to get this message in the log file as well.